lib/simplecov/result_merger.rb

```diff
@@ -54,20 +54,24 @@ def results
         results
       end
 
+      # Merge two or more SimpleCov::Results into a new one with merged
+      # coverage data and the command_name for the result consisting of a join
+      # on all source result's names
+      def merge_results(*results)
```

Comment:
@PragTob I listed a few in https://github.com/colszowka/simplecov/pull/558#pullrequestreview-21775132